docs/how-to-guides/add-tailwind.md

```diff
@@ -0,0 +1,53 @@
+---
+myst:
+  html_meta:
+    "description": "How to add Tailwind CSS to your add-on"
+    "property=og:description": "How to add Tailwind CSS to your add-on"
+    "property=og:title": "How to add Tailwind CSS to your add-on"
+    "keywords": "Seven, frontend, Plone, how-to, theming, Tailwind CSS, tailwind"
+---
+
+# How to add Tailwind CSS to your add-on
+
+This guide will walk you through the steps to integrate Tailwind CSS into your add-on.
+
+## Install Tailwind CSS in your add-on
+
+```shell
+pnpm --filter <addon-name> add tailwindcss
```

Comment:
```suggestion
pnpm --filter <add-on-name> add tailwindcss
```

Reply:
Do you think this is necessary? In lieu of being less verbose, at least. And, that we will have to change it in everything that I wrote yesterday.

Reply:
Unfortunately, yes. After all, it's the name of an "add-on". I caught them all, so add to batch should be fine.